src/Instance/InstanceDetails/InstanceElecAccessView/InstanceElecAccessView.js

```diff
@@ -0,0 +1,107 @@
+import React, {
+  useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import {
+  Accordion,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import {
+  checkIfArrayIsEmpty,
+} from '../../../utils';
+import {
+  noValue,
+  wrappingCell,
+} from '../../../constants';
+
+const visibleColumns = ['URL relationship', 'URI', 'Link text', 'Materials specified', 'URL public note'];
+const columnWidths = {
+  'URL relationship': '25%',
+  'URI': '25%',
+  'Link text': '25%',
+  'Materials specified': '25%',
+  'URL public note': '25%',
+};
+const getColumnMapping = intl => ({
+  'URL relationship': intl.formatMessage({ id: 'ui-inventory.URLrelationship' }),
```

Comment:
Can the keys like specified in translation be used as keys?
```suggestion
  URLrelationship: intl.formatMessage({ id: 'ui-inventory.URLrelationship' }),
```

Reply:
I don't think as this key is object property is passed to rows

Reply:
I think it should work just fine. As mapping to translations values is done in `getColumnMapping` function itself. But keys that I suggested that should be reflected in `visibleColumns`, `columnWidths`.

Reply:
ah I see what you mean, but in this case formatter will have these keys, so no much sense here IMO